addons/l10n_ke_edi_tremol/static/src/js/send_invoice.js

```diff
@@ -6,28 +6,43 @@ odoo.define('l10n_ke_edi_tremol.action_post_send_invoice', function (require) {
     var _t = core._t;
 
     async function post_send(parent, {params}) {
-        const move_id = params.move_id;
-        await ajax.post(params.proxy_address + '/hw_proxy/l10n_ke_cu_send', params).then(function (res) {
-            const res_obj = JSON.parse(res);
-            if (res_obj.status != "ok") {
-                Dialog.alert(this, "Posting the invoice has failed, with the message: \n" + res_obj.status);
-            } else {
-                rpc.query({
-                    model: 'account.move',
-                    method: 'l10n_ke_cu_response',
-                    args: [[], {'replies': res_obj.replies, 'serial_number': res_obj.serial_number, 'move_id': move_id}],
-                }).then(function () {
-                    parent.services.action.doAction({
-                        'type': 'ir.actions.client',
-                        'tag': 'reload',
-                    });
-                }, function () {
-                    Dialog.alert(this, _t("Error trying to connect to Odoo. Check your internet connection"));
-                })
+        let refresh = false;
+        for (let move_id in params.invoices) {
+            try {
+                const res = await ajax.post(
+                    params.invoices[move_id].proxy_address + '/hw_proxy/l10n_ke_cu_send', {
```

Comment:
maybe as params it is easier to do params.invoices[move_id]

Reply:
I think it's better to send a smaller request, with less redundant data

Reply:
this is probably actually better than before because we only send the necessary parameters